### PR TITLE
Add to StorIO.LowLevel getter for underlying sqLiteOpenHelper

### PR DESCRIPTION
**Pull request by nikitin-da**
Part of #704

### CODE REVIEW

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/StorIOSQLite.java [hunk 1]
```diff
@@ -329,6 +330,21 @@
          * how to use this and when transactions are committed and rolled back.
          */
         public abstract void endTransaction();
+
+        /**
+         * Returns {@link SQLiteOpenHelper} for the direct access to underlined database.
```
Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/StorIOSQLite.java]: underlined -> underlying

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/StorIOSQLite.java [hunk 2]
```diff
@@ -329,6 +330,21 @@
...
+         * It can be used in cases when {@link StorIOSQLite} support doesn't provided.
```
Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/StorIOSQLite.java]: `when {@link StorIOSQLite} APIs are not enough.` Sounds better to me :)

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/StorIOSQLite.java [hunk 3]
```diff
@@ -329,6 +330,21 @@
...
+         * <p>
+         * Notice: Database changes through the direct access
+         * to the {@link SQLiteOpenHelper} will not trigger notifications.
+         * If it possible you should use {@link StorIOSQLite} methods instead
+         * or call {@link #notifyAboutChanges(Changes)}.
```
Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/StorIOSQLite.java]: `or call … manually.`

File: storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/design/DesignTestStorIOSQLite.java [hunk 4]
```diff
@@ -94,6 +95,14 @@ public void setTransactionSuccessful() {
         public void endTransaction() {
             // no impl
         }
+
+        @NonNull
+        @Override
+        public SQLiteOpenHelper sqliteOpenHelper() {
+            // doesn't required in design test
```
Comment (artem-zinnatullin) [inline storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/design/DesignTestStorIOSQLite.java]: Not required in design test.

File: storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLiteTest.java [hunk 5]
```diff
@@ -69,6 +69,15 @@ public void nullSQLiteOpenHelper() {
     }
 
     @Test
+    public void lowLevelReturnsSameInstanceOfSQLiteOpenHelper() {
```
Comment (artem-zinnatullin) [inline storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLiteTest.java]: 👍

**Comment by nikitin-da** [comment]
@geralt-encore PTAL, please)

**Pull request merged**